doc/manual/requirements.md

```diff
@@ -27,7 +27,9 @@ Please consider using a virtual machine to run Huginn on Windows.
 
 ## Ruby versions
 
-Huginn requires Ruby (MRI) 2.2 or 2.3.
+Huginn requires Ruby (MRI) >= 3.2.2.
```

Comment:
We are currently at Ruby 3.2.4 and will receive teeny updates in the future, so please leave out the teeny version here.
```suggestion
Huginn requires Ruby (MRI) >= 3.2.
```